### ISSUE TITLE
Last upate broke my entire configuration

### ISSUE THREAD

**Issue by nachovizzo** [regolith-linux/regolith-desktop#328]
The last release completely broke my entire configuration. I'm trying to downgrade it. is there any quick way of doing it?

I will share details of the errors whenever my system becomes functional again!

**Comment by kgilmer** [comment]
I'm guessing you're getting packages from ppa:regolith-linux/stable as that was updated yesterday.  You can change to the less-often-updated ppa:regolith-linux/release which is still set at R1.3.1, or go to ppa:regolith-linux/r1.3 which hasn't changed since the R1.3 release, depending on how far back your system is based.

**Comment by kgilmer** [comment]
you'll need to either re-install Regolith after changing the PPA, or maybe [something like this](https://askubuntu.com/a/201622/834410) would work (I haven't tested it.)

**Comment by nachovizzo** [comment]
I see, thanks!

For now `sudo apt install regolith-desktop=2.49-1ubuntu1\~ppa1` and `sudo apt install regolith-i3xrocks-config=2.2.14-1ubuntu1\~ppa1` gave me back port of the interface.

Is there any documentation on how to update to r.1.4 I don't mind at all to report bugs and whatsoever ;)

I have a really custom configuration https://github.com/nachovizzo/dotfiles/tree/master/.config/regolith

I tried to merge changes from the /etc/regolith files, but nothing worked. One thing I've noticed is that the package manager didn't install any of the i3xocks scripts.

How can i help!? Thanks!
